benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go

```diff
@@ -17,6 +17,7 @@ package kubectl
 import (
 	"context"
 	"fmt"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/test/utils/log"
```

Comment:
I guess this reordering of import log is by accident.

Reply:
yeah